net-print/gutenprint/gutenprint-5.3.4.ebuild

```diff
@@ -0,0 +1,88 @@
+# Copyright 1999-2021 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+inherit autotools
+
+DESCRIPTION="Ghostscript and cups printer drivers"
+HOMEPAGE="http://gutenprint.sourceforge.net"
+
+MY_P="${P/_/-}"
+S="${WORKDIR}/${MY_P}"
+SRC_URI="mirror://sourceforge/gimp-print/${MY_P}.tar.xz"
+
+LICENSE="GPL-2"
+SLOT="0"
+KEYWORDS="~alpha ~amd64 ~arm ~hppa ~ia64 ~ppc ~ppc64 ~sparc ~x86"
+
+IUSE="cups gimp gtk nls readline ppds static-libs"
+RESTRICT="test"
+REQUIRED_USE="gimp? ( gtk )"
+
+BDEPEND="
+	virtual/pkgconfig
+	nls? ( sys-devel/gettext )
+"
+
+# gimp restriction: https://sourceforge.net/p/gimp-print/bugs/725/
+RDEPEND="
+	dev-lang/perl
+	cups? ( >=net-print/cups-1.1.14 )
+	gimp? (
+		media-gfx/gimp:0/2
+		x11-libs/gtk+:2
+	)
+	gtk? ( x11-libs/gtk+:2 )
+	readline? ( sys-libs/readline:0= )
+"
+
+BDEPEND="
+	${RDEPEND}
+	nls? ( virtual/libintl )
```

Comment:
Should this be `DEPEND` instead? Is virtual/libintl needed in DEPEND too? 
`BDEPEND="${RDEPEND}"` is almost always wrong though.